website/next-i18next.config.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
       "vi",
       "zh",
       "tr",
+      "fa",
```

Comment:
I see we're not totally ordering these alphabetically.  Could you put this as alphabetically as you can?